source/publishing_data/11_saving_and_publishing_a_dataset/dataset_actions_status.rst

```diff
@@ -0,0 +1,69 @@
+Dataset actions and statuses
+============================
+
+Actions
+~~~~~~~
+
+**Actions related to the life cycle of a dataset**
+
+.. list-table::
+  :header-rows: 1
+
+  * * Action
+    * Button
+    * Description
+  * * Save
+    * .. image:: images/button_save.png
+    * Saves all new modifications, processings and configurations operated on the dataset.
```

Comment:
I would suggest: "Saves all new configuration modifications operated on the dataset." 
In my opinion processing could be confusing.